content/blog/hello-world/index.md

```diff
@@ -1,233 +1,8 @@
 ---
 title: Hello World
-date: "2015-05-01T22:12:03.284Z"
+date: "2021-11-22T14:52:03.284Z"
 description: "Hello World"
-authors: [{lastname: "Bar",
-           firstname: "Foo"}]
+authors: [{lastname: "admin",
```

Comment:
I would like to see a 'real' name here, because the author is shown beneath the blog post title
![image](https://user-images.githubusercontent.com/72247962/142881125-77d1d55e-6c63-414b-b973-0d37e983a5f2.png)

Reply:
Ok, did that. Merged and deployed.